scripts/sct_download_data.py

```diff
@@ -88,8 +88,8 @@ def main(args=None):
     dict_url = {
         'sct_example_data': ['https://osf.io/kjcgs/?action=download',
                              'https://www.neuro.polymtl.ca/_media/downloads/sct/20180525_sct_example_data.zip'],
-        'sct_testing_data': ['https://osf.io/yutrj/?action=download',
-                             'https://www.neuro.polymtl.ca/_media/downloads/sct/20191031_sct_testing_data.zip'],
+        'sct_testing_data': ['https://osf.io/63wba/?action=download',
```

Comment:
pls indicate the changes in the `README.md` file (root of the folder)

Reply:
fixed in https://github.com/neuropoly/spinalcordtoolbox/pull/2679/commits/2eda8a8164458d46e3ed17d3d70c1336858c45a7

Reply:
I updated both file on osf & wiki

Reply:
just looked at "https://osf.io/6x5a2/?action=download" and the change is still not mentioned in the readme:

~~~
sct_example_data
================

Example data for the Spinal Cord Toolbox (errsm_30)

2019-10-31 (CG)
- correct template/template/info_label.txt IDs

2016-11-22
- updated info_label

2016-10-05
- added t2s
~~~

also, if you could use the same URL but just replace the file it would be better. OSF has a versioning system. I would prefer to avoid having people deleting files from the OSF site. it is too dangerous. thx

Reply:
done. I initially changes the README.md inside the template folder.

Reply:
no, please follow the chronological logic.
also: add your initials in brackets (as done with "CG").
also: let's add a blurb for future modifs, and create a separate CHANGES.md file.

to simplify, i'll send you the files (cannot upload here for some reasons...)

Reply:
I uploaded the new files.